app/javascript/components/pages/FateNew.js

```diff
@@ -15,68 +14,127 @@ class FateNew extends Component{
         description: "",
         is_public: false,
       },
-      submitted: false
-    }
+      submitted: false,
+      urls: [],
+    };
   }
 
   handleChange = (e) => {
-        // destructuring form out of state
-        const { form } = this.state
-        form[e.target.name] = e.target.value
-        // setting state to the updated form
-        this.setState({ form: form })
-    }
-    handleSubmit = (e) => {
-        e.preventDefault()
-        this.props.createNewEvent(this.state.form)
-        this.setState({submitted: true})
-    }
+    // destructuring form out of state
+    const { form } = this.state;
+    form[e.target.name] = e.target.value;
+    // setting state to the updated form
+    this.setState({ form: form });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.props.createNewEvent(this.state.form);
+    this.setState({ submitted: true });
+  };
 
-    render() {
+  fetchImages = async () => {
+    const response = await fetch(
```

Comment:
We should check there's actually a value for category before we even try to make the request.

Reply:
fixed

Reply:
fixed